### DISCUSSION TITLE
Not possible to run medusa-cli without global install?

### DISCUSSION THREAD

**Discussion by bjornarhagen** [medusajs/medusa#4432] [Q&A] [answered]
Usually I am able to install CLI tools and other npm packages as a dev dependency and run them from `./node_modules/.bin/`. That way I avoid filling up my system with all these different tools, plus I get to have different versions in different projects.

For example:
1. `npm install next --save-dev`
2. `./node_modules/.bin/next -v`
Output: `Next.js v13.4.8-canary.2`

But trying this with the Medusa CLI this happens:

```
> ./node_modules/.bin/medusa -v
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module './dist/bin/medusa.js'
```

Is this a bug? Or is not supported due to some technical limitations?

**Comment by olivermrbl** [comment, marked as answer]
I cannot seem to reproduce your issue. Can I get you to ensure `@medusajs/medusa` and `@medusajs/medusa-cli` are installed?

My output is:
```
node_modules/.bin/medusa -v
1.12.2-next-20230626114329
```

(I am running the next version)

**Reply by bjornarhagen** [reply]
🤦‍♂️ Sorry my bad, yeah I only had `@medusajs/medusa-cli` installed. Adding `@medusajs/medusa` fixed it. Thank you!
